README.md

```diff
@@ -1,8 +1,248 @@
-# BraveNewAlgorithm
+# BraveNewAlgorithm.jl
 
-This is a metaheuristic that is inspired by Brave New World and its specific caste system and way of reproduction. It tries to improve the exploitation/exploration balance of population-based metaheuristics by using different castes for every purpose.
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 
-If you want to use it, we kindly ask you to cite the following paper:
+A metaheuristic optimization algorithm inspired by Aldous Huxley's "Brave New World" and its caste system. This algorithm improves the exploration/exploitation balance of population-based metaheuristics by using different castes with distinct roles in the evolutionary process.
+
+## 🚀 Quick Start
+
+```bash
+# Clone the repository
+git clone https://github.com/cecimerelo/BraveNewAlgorithm.jl.git
+cd BraveNewAlgorithm.jl
+
+# Install dependencies (this may take a few minutes)
+julia examples/setup.jl
+# OR use the Makefile: make instantiate
+
+# Run the quick start example
+julia examples/quickstart.jl
+```
+
+## 📦 Installation
+
+### From GitHub (Recommended)
```

Comment:
Why is this recommended? It's also included in the Julia registry, although it might not be the current version.